vnext/Microsoft.ReactNative.Cxx/StructInfo.h

```diff
@@ -23,12 +34,25 @@
     fieldMap.emplace(fieldName, &TClass::field);                           \
   }
 
-#define INTERNAL_REACT_FIELD_1_ARGS(field) INTERNAL_REACT_FIELD_2_ARGS(field, L## #field)
-#define INTERNAL_REACT_FIELD_3RD_ARG(arg1, arg2, arg3, ...) arg3
-#define INTERNAL_REACT_FIELD_RECOMPOSER2(argsWithParentheses) INTERNAL_REACT_FIELD_3RD_ARG argsWithParentheses
+#define INTERNAL_REACT_FIELD_1_ARG(field) INTERNAL_REACT_FIELD_2_ARGS(field, L## #field)
 #define INTERNAL_REACT_FIELD(...) \
-  INTERNAL_REACT_FIELD_RECOMPOSER2((__VA_ARGS__, INTERNAL_REACT_FIELD_2_ARGS, INTERNAL_REACT_FIELD_1_ARGS, ))
-
+  INTERNAL_REACT_RECOMPOSER_3((__VA_ARGS__, INTERNAL_REACT_FIELD_2_ARGS, INTERNAL_REACT_FIELD_1_ARG, ))
+
+// REACT_STRUCT(structType)
```

Comment:
For my education: Is this a known document format (e.g. something Doxygen supports)?

Reply:
No, it is not supported by Doxygen. It just makes more visible in the Visual Studio IntelliSense.

---
In reply to: [406953385](https://github.com/microsoft/react-native-windows/pull/4568#discussion_r406953385) [](ancestors = 406953385)